l10n_es_aeat_sii/models/account_invoice.py

```diff
@@ -149,7 +153,7 @@ def onchange_fiscal_position_l10n_es_aeat_sii(self):
             else:
                 key = invoice.fiscal_position.sii_registration_key_purchase
             invoice.sii_registration_key = key
-
+    
```

Comment:
https://stackoverflow.com/questions/7744258/how-to-remove-trailing-whitespace-in-pydev-plugin-for-eclipse